src/App.js

```diff
@@ -1,24 +1,46 @@
 import React from 'react';
-import './App.css';
+import { Tab } from './components/Tab/Tab';
+import 'bulma/css/bulma.css';
 
-// eslint-disable-next-line no-unused-vars
 const tabs = [
   {
-    title: 'Tab 1',
-    content: 'Some text 1',
+    title: 'Tab 1', content: 'Some text 1',
   },
   {
-    title: 'Tab 2',
-    content: 'Some text 2',
+    title: 'Tab 2', content: 'Some text 2',
   },
   {
-    title: 'Tab 3',
-    content: 'Some text 3',
+    title: 'Tab 3', content: 'Some text 3',
   },
 ];
 
-const App = () => (
-  <h1>React tabs</h1>
-);
+class App extends React.Component {
+  state = {
+    active: this.selectTab,
```

Comment:
I meant not this)) now you save object method in state, but I meant to save active tab (object) or it's index (number) in App's state